packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.controller.js

```diff
@@ -217,13 +226,26 @@ class LogsHomeCtrl {
         loaderFunction: () =>
           this.LogsTokensService.getDefaultCluster(this.serviceName),
       });
-      this.storageData = this.CucControllerHelper.request.getHashLoader({
+      this.streamData = this.CucControllerHelper.request.getHashLoader({
+        loaderFunction: () =>
+          this.LogsHomeService.getDataUsage(
+            this.serviceName,
+            this.LogsConstants.DATA_STORAGE.METRICS.STREAM_SIZE,
+          ),
+      });
+      this.archiveData = this.CucControllerHelper.request.getHashLoader({
         loaderFunction: () =>
-          this.LogsHomeService.getDataUsage(this.serviceName),
+          this.LogsHomeService.getDataUsage(
+            this.serviceName,
+            this.LogsConstants.DATA_STORAGE.METRICS.COLD_STORAGE_TOTAL,
+          ),
       });
-      this.coldStorage = this.CucControllerHelper.request.getHashLoader({
+      this.indiceData = this.CucControllerHelper.request.getHashLoader({
```

Comment:
We should try avoiding using this helper and use Promises
something like 
```suggestion
      this.LogsHomeService.getDataUsage(
            this.serviceName,
            this.LogsConstants.DATA_STORAGE.METRICS.INDEX_SIZE,
          )
          .then((indiceData) => {
            this.indiceData = indiceData
           })
```
I understand given the context that it could require some work so it's ok but let's keep that in mind :)

Reply:
Tried quickly, without success.